webpack.common.js

```diff
@@ -0,0 +1,66 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const ManifestPlugin = require('webpack-pwa-manifest')
+const { WebpackManifestPlugin } = require('webpack-manifest-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+module.exports = {
+  entry: {
+    index: './src/index.tsx',
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.(ts|js)x?$/,
+        include: path.resolve(__dirname, 'src'),
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: [
+              '@babel/preset-env',
+              '@babel/preset-react',
+              '@babel/preset-typescript',
+            ],
+          },
+        },
+      },
+    ],
+  },
+  resolve: {
+    extensions: [ '.tsx', '.ts', '.jsx', '.js' ],
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './public/index.html',
+      favicon: './public/favicon.svg',
+      filename: './index.html',
+    }),
+    // Serves the manifest.json file
+    new ManifestPlugin({
+      filename: "manifest.json",
+      name: 'Your App Name Here',
+      short_name: 'App Name',
+      description: 'An amazing TeachLA React Website :D',
+      background_color: '#ffffff',
+      theme_color: '#000000',
+      icons: [
+        {
+          src: path.resolve('./public/favicon512.png'),
+          sizes: [192, 512] // multiple sizes
+        },
```

Comment:
Does this file exist? No, right?

Reply:
I think it does? Also i think the plugin helps resize it so that the different image sizes exist?
![image](https://user-images.githubusercontent.com/29292203/121831731-0746f080-cc7d-11eb-9134-050585fdc1c3.png)
 
idk if this is best practice tho? would you prefer using this plugin or having different sized favicons?

Reply:
Oh I didn't realize that the plugin makes the PNG from the SVG - cool! (it does, right?)

Reply:
i have a `favicon512.png`! and it helps resize it to 192

Reply:
Oh shoot, totally missed that - thanks!